### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by jasmineherrera**
Added project 2 of week 2 with the form to input contact info.

### CODE REVIEW

**Comment by jasmineherrera** [comment]
Sorry Kelly this will be the last pull request for week 2 work!

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    // Get the value entered in the form.
+    String nameValue = request.getParameter("name-input");
+    String emailValue = request.getParameter("email-input");
+    String messValue = request.getParameter("mess-input");
+
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("User submitted:\nName = " + nameValue + "\nEmail = " + emailValue + "\nMessage = " + messValue);
+
```
Comment (ngkelly3) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:23]: Do you still need the print statement? You can delete it if you don't need it anymore.

**Review by ngkelly3 — CHANGES_REQUESTED** [review]
Hey Jasmine, looks good, just wondering if you want to remove that print statement before submitting or if you need it for later?

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java (hunk 1 above)
Comment (jasmineherrera) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:23]: I'll probably keep it just so I can keep track of the inputs from the backend

Comment (ngkelly3) [reply portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:23]: Okay sgtm!

**Review by ngkelly3 — APPROVED** [review]

**Pull request merged**